--- sinjdoc-0.5/src/main/Main.java~	2003-08-01 03:14:10.000000000 +0200
+++ sinjdoc-0.5/src/main/Main.java	2007-07-26 23:08:13.000000000 +0200
@@ -320,9 +320,9 @@
 	addOption(new Option("-doclet", "<class>", 2,
 			     "Generate output via alternate doclet") {
 		void process(RunData rd, List<String> args) {
-		    if (rd.doclet!=null)
-			throw new RuntimeException
-			    ("-doclet given after doclet option.");
+		    //if (rd.doclet!=null)
+			//throw new RuntimeException
+			//    ("-doclet given after doclet option.");
 		    rd.defaultDocletClassName=args.get(1);
 		}
 	    });
